pages/common/erl.md

```diff
@@ -0,0 +1,15 @@
+# erl
+
+> Run and manage programs in Erlang programming language.
+
+- Run sequential Erlang program as a common script and then exit:
+
+`erlc *.erl && erl -noshell '{{mymodule:myfunction(arguments)}}, init:stop().'`
+
+- Connect to a running Erlang node:
+
+`erl -remsh {{nodename}}@{{hostname}} -sname {{custom_shortname}} -hidden -setcookie {{cookie_of_remote_node}}`
+
+- Tell the Erlang shell to load another modules or version:
```

Comment:
Not sure that `modules` needs to be plural here.

Reply:
I believe that yes as there is given a path to the directory where multiple files could be found

Reply:
I see. In that case, we'll need to rework it a bit. How about ....`to load modules from a directory:`?

Reply:
Thanks, that sounds good.